src/Form/test-cases/bitwarden_com_account-locked.html

```diff
@@ -0,0 +1,36 @@
+<form ngnativevalidate="" class="container ng-touched ng-dirty ng-valid ng-submitted">
+    <div class="row justify-content-md-center mt-5">
+        <div class="col-5">
+            <p class="text-center mb-4">
+                <i aria-hidden="true" class="bwi bwi-lock bwi-4x text-muted"></i>
+            </p>
+            <p class="lead text-center mx-4 mb-4">Your vault is locked. Verify your master password to continue.</p>
+            <div class="card d-block">
+                <div class="card-body">
+                    <div class="form-group">
+                        <label for="masterPassword">Master password</label>
+                        <div class="d-flex">
+                            <input id="masterPassword" name="MasterPassword" required="" appautofocus="" appinputverbatim="" class="text-monospace form-control ng-touched ng-dirty ng-valid" type="password" autocomplete="off" autocapitalize="none" autocorrect="none" spellcheck="false" inputmode="verbatim" data-manual-scoring="password">
+                            <button type="button" class="ml-1 btn btn-link" title="Toggle visibility" aria-label="Toggle visibility">
+                                <i aria-hidden="true" class="bwi bwi-lg bwi-eye"></i>
+                            </button>
+                        </div>
+                        <small class="text-muted form-text">
+                            Logged in as emanuele@duckduckgo.com on bitwarden.com.
```

Comment:
NAB: Do you want your email in the codebase? 😁

Reply:
LOL, I do not! Thanks for spotting this.